### PR TITLE
[KYUUBI #5259][UI] Web UI supports Swagger pages

### PR DESCRIPTION
**Pull request by zwangsheng**
### _Why are the changes needed?_

As title, add swagger ui into new kyuubi web ui.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/contributing/code/testing.html#running-tests) locally before make a pull request

### _Was this patch authored or co-authored using generative AI tooling?_

No

### CODE REVIEW

**Comment by zwangsheng** [comment]
Current, we can use swagger ui in new web-ui, but css seems have some issue, cc @cfmcgrady WIP.

截屏2023-09-08 16 31 26

File: kyuubi-server/web-ui/src/views/swagger/index.vue [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+* limitations under the License.
+-->
+
+<template>
+  <main>
+    Swagger
+    <div id="swagger-ui" class="swagger-ui"> </div>
+  </main>
+</template>
+<script lang="ts">
```
Comment (cfmcgrady) [inline kyuubi-server/web-ui/src/views/swagger/index.vue:25]:
```suggestion
<script lang="ts">
  import 'swagger-ui-dist/swagger-ui.css';
```

**Comment by cfmcgrady** [comment]
> Current, we can use swagger ui in new web-ui, but css seems have some issue, cc @cfmcgrady WIP.
> 
> 截屏2023-09-08 16 31 26

it works. :)

截屏2023-09-11 上午11 17 31

**Comment by zwangsheng** [comment]
@cfmcgrady Thanks for your help !!!!

I will finish the rest work.

File: kyuubi-server/web-ui/src/views/swagger/index.vue [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  import 'swagger-ui-dist/swagger-ui.css';
```
Comment (pan3793) [inline kyuubi-server/web-ui/src/views/swagger/index.vue:26]:
```suggestion
  import 'swagger-ui-dist/swagger-ui.css'
```

**Comment by pan3793** [comment]
don't forget to update the `NOTICE-binary`

**Comment by zwangsheng** [comment]
Ignore this background color.
截屏2023-09-11 15 38 42

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merged to master/1.8

**Pull request closed without merging**